src/element/shape.js

```diff
@@ -598,39 +598,40 @@ export class ShapeElement extends Element {
       this.isClosed = shapePath.isClosed;
       const paths = shapePath.path.paths;
       if (frame < paths[0].startFrame) {
+        if (index !== 0 && Number(PIXI.VERSION.charAt(0)) === 5) { this.beginHole(); }
```

Comment:
I try to add https://github.com/knocknote/pixi-after-effects/pull/12

Could you resolve by using the above feature ?

Reply:
Thanks for your help @goccy I have just commited the shape.js with your annotations! 😄

Reply:
> May you resolve de conflict with the dist file?

Yes !! Please resolve conflict that file !